dataans/src-tauri/src/dataans/mod.rs

```diff
@@ -72,10 +72,10 @@ impl DataansState {
     }
 }
 
-pub fn init_dataans_plugin<R: Runtime>() -> TauriPlugin<R> {
+pub fn init_dataans_plugin<R: Runtime>() -> TauriPlugin<R, ()> {
     debug!("init_dataans_plugin");
 
-    Builder::new(APP_PLUGIN_NAME)
+    Builder::<R, ()>::new(APP_PLUGIN_NAME)
```

Comment:
why did you change the type parameters? did you have any compilation errors with the old code?